batch.R

```diff
@@ -170,8 +170,8 @@ for(i in 1:nrow(fileDF)) {
   #TODO: model metrics for non-rloadest models
   #combine into DF with row for each model
   #need to extract fitted model so rloadest functions can be used
-  metrics <- summarizeModel(getFittedModel(rloadest5param))
-  write.csv(x = metrics, file = file.path(outputFolder, constitName, "modelMetrics", paste(constitSite, "modelMetrics.csv", sep = "_")), row.names = FALSE)
+  metrics <- summarizeModel(getFittedModel(rloadest5param), constituent = constitName, site = constitSite)
```

Comment:
for this PR and for https://github.com/USGS-R/loadflex/pull/169, can I propose a different approach? in this line, use
```
metrics <- summarizeModel(rloadest5param)
```
instead. Then I hope we could use `summarizeModel.loadReg2` to attach information that's present in the loadflex model metadata but not in the rloadest model object. That info can go here: https://github.com/USGS-R/loadflex/blob/master/R/loadReg2.R#L475

Would that eliminate the need to have additional arguments for `constituent` and `site` here? I would like to avoid them if possible, because they introduce redundancy and therefore increase the probability that a user someday creates a summary with the wrong constituent and site labels.